auth/auth.go

```diff
@@ -25,7 +54,10 @@ func Auth(rw http.ResponseWriter, req *http.Request) bool {
 		return false
 	}
 
-	if token.Claims["hostUuid"] != config.Config.HostUuid {
+	if token.Claims["reportedUuid"] != nil && token.Claims["reportedUuid"] != config.Config.HostUuid {
+		return false
+	}
+	if token.Claims["hostUuid"] != nil && token.Claims["hostUuid"] != config.Config.HostUuid {
```

Comment:
Again, because of the corresponding cattle change where the key was being passed as "ReportedUuid". Changed it back to original condition.